src/cdk/drag-drop/drag-utils.ts

```diff
@@ -51,6 +50,28 @@ export function transferArrayItem<T = any>(currentArray: T[],
   }
 }
 
+/**
+ * Copies item from one array to another.
+ *
+ * https://github.com/angular/material2/blob/master/src/cdk/drag-drop/drag-utils.ts
```

Comment:
Can you get rid of this link? Also the description should mention that it keeps the item in its original position in the source array.